glossary/Git.md

```diff
@@ -0,0 +1,4 @@
+### Git
+
+Git is an open-source version control system, used for source code management.
+Git allows users to clone and edit repositories on their local machines, without having to change the master repository.
```

Comment:
this is pretty vague... I would reword it 😅  also, I would change the last sentence to "the remote master repository"

Reply:
Honestly, it's based on Wikipedia and other sources. It's the least vague I could write. Could you just reword it on the same branch? Maybe you can explain it better than that.

Reply:
Updated. Tried to keep the terminology present as it could be of use to beginners.